petpals/messaging/static/js/conversation_message.js

```diff
@@ -0,0 +1,91 @@
+var submitButton = document.getElementById('submit');
```

Comment:
There is no submit button. You should probably add one.

Reply:
Removed.